editor/animation_bezier_editor.cpp

```diff
@@ -34,6 +34,7 @@
 #include "editor_scale.h"
 #include "scene/gui/view_panner.h"
 #include "scene/resources/text_line.h"
+#include <limits.h>
```

Comment:
Nitpick but please keep Godot includes and system includes separate:
```suggestion
#include "editor_scale.h"
#include "scene/gui/view_panner.h"
#include "scene/resources/text_line.h"

#include <limits.h>
```